docs/cli/kubectl-testkube_delete_webhook.md

```diff
@@ -21,9 +21,11 @@ kubectl-testkube delete webhook <webhookName> [flags]
 ### Options inherited from parent commands
 
 ```
-      --analytics-enabled   enable analytics (default true)
+      --analytics-enabled   enable analytics
+  -w, --api-uri string      api uri, default value read from config if set
```

Comment:
`-w` could be a little counter intuitive

Reply:
a - should be better)